query-service/src/integrationTest/resources/configs/query-service/application.conf

```diff
@@ -0,0 +1,276 @@
+main.class = org.hypertrace.core.query.service.QueryServiceStarter
```

Comment:
How are we planning on keeping this file updated? 🤔 I briefly remember this came up in some PR

Reply:
Oh, it's the same PR

Reply:
Why do we need a copy though? Since, this in an integration which would be bringing up query service, it would directly read the `application.conf` https://github.com/hypertrace/query-service/blob/main/query-service/src/main/resources/configs/common/application.conf

Or, am I missing something?

Reply:
Removed, had to make some changes in integration test framework (which is why I had added this extra config, and most other repos follow the same pattern)